app/assets/javascripts/components/nav.jsx

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
+import { Link } from 'react-router';
+import { slide as Menu } from 'react-burger-menu';
+const Nav = React.createClass({
+  displayName: 'Nav',
+
+  getInitialState() {
+    const rootUrl = $('#nav_root').data('rooturl');
+    const logoPath = $('#nav_root').data('logopath');
+    const fluid = $('#nav_root').data('fluid');
+    const exploreurl = $('#nav_root').data('exploreurl');
+    const explorename = $('#nav_root').data('explorename');
+    const exploreclass = $('#nav_root').data('exploreclass');
+    const usersignedin = $('#nav_root').data('usersignedin');
+    const ifadmin = $('#nav_root').data('ifadmin');
+    const trainingurl = $('#nav_root').data('trainingurl');
+    const disableTraining = $('#nav_root').data('disable_training');
+    const helpDisabled = $('#nav_root').data('help_disabled');
+    const askUrl = $('#nav_root').data('ask_url');
```

Comment:
@ragesoss i have doubts related to this, about how to send data to a new window opened from react.